data-es6.js

```diff
@@ -3570,8 +3591,9 @@ exports.tests = [
         var obj = { foo: 1, bar: 2 };
         var iterator = Reflect.enumerate(obj);
 
+        var passed = Symbol.iterator in item;
```

Comment:
I thought about it, but currently only IE TP adds `Reflect.enumerate`. Problem can be only if es6-shim will add `Reflect` module without `Symbol`. Ok,  I will add check.